src/components/editor/loop.vue

```diff
@@ -3,7 +3,7 @@
     <draggable
       style="min-height: 80px;"
       v-model="items"
-      :options="{group: {name: 'controls'}}"
+      v-bind="{group: {name: 'controls'}}"
```

Comment:
I think using a single prop reads better.
```suggestion
      group="controls"
```

Looking at the docs (https://github.com/SortableJS/Vue.Draggable/blob/master/documentation/migrate.md#options-props), it seems that the above should be valid.

Reply:
the changes were made